### DISCUSSION TITLE
Please remove or reduce the limit on the size of files that can be stored in storage.

### DISCUSSION THREAD

**Discussion by happy-spark** [community/community#27728] [Other Feature Feedback, Questions, & Ideas] [answered]
**The context may be strange because I used a translator😅**

Remove or reduce the limit on the size of files that can be stored in storage.

**Comment by jfullstackdev** [comment, marked as answer]
if it's in Windows try WinRAR

**Reply by happy-spark** [reply]
Thank you very much for the good feedback.
